NEXT_CHANGES.rst

```diff
@@ -0,0 +1,200 @@
+Instructions
+============
+
+Use this as a staging ground for CHANGES.rst. In other words, describe the
+changes and additions to ixdat's API associated with your contribution. The idea is
+that what you write here informs other developers what is on its way and then will be
+copied to CHANGES.rst when the next version of ixdat is distributed. Please include
+links to relevant Issues, Discussions, and PR's on github with the following format
+(replace XX):
+
+`Issue #XX <https://github.com/ixdat/ixdat/issues/XX>`_
+
+`PR #XX <https://github.com/ixdat/ixdat/pulls/XX>`_
+
+Coming changes for ixdat 0.2.0
+==============================
+
+API Changes
+-----------
+
+ixdat.measurement
+^^^^^^^^^^^^^^^^^
+
+- Generalization of multiple calibrations:
+
+  ``Measurement.calibration`` is deprecated in favor of ``Measurement.calibration_list``.
+  A dummy calibration property is put in with a getter and setter which raise
+  ``DepreciationWarning``'s. Use, e.g.:
+
+  - ``meas.calibration_list[0]`` (replaces ``meas.calibration``).
+
+  - ``meas.add_calibration(my_cal)`` (replaces ``meas.calibration = my_cal``).
+
+- New method ``Measurement.calibrate(*args, **kwargs)``
+
+  ``meas.calibrate(...)`` is a shortcut for ``meas.add_calibration(calibration_class(...))``
+  where calibration_class is found by looking up ``meas.techniques`` in
+  ``ixdat.technqiues.CALIBRATION_CLASSES``. For esample, if ``meas`` is an
+  ``ECMSMeasurement``, calibration_class is ``ECMSCalibration``.
+
+  Together with the generalization of multiple calibrations, this enables very flexible
+  calibration. All of the following code examples work.
+
+  1. When measurements are appended or hyphenated, all their calibrations carry over.
+  This example results in ``my_ecms_meas.calibration_list`` having an
+  ``ECCalibration`` and an ``MSCalibration``, both of which are accessible to ``grab``
+  and plotting functions.
+
+    >>> my_ec_meas.calibrate(RE_vs_RHE=0.715, A_el=0.196)
+    >>> my_ms_meas.calibrate(ms_cal_results=[my_H2_at_M2, my_O2_at_M32])
+    >>> my_ecms_meas = my_ec_meas + my_ms_meas
+    >>> my_ecms_meas.plot(mol_list=["H2", "O2"])
+
+  2. You can calibrate one measurement multiple times. When two calibrations contain the
+  same parameter, the last one added is used:
+
+    >>> my_ecms_meas = my_ec_meas + my_ms_meas
+    >>> my_ecms_meas.calibrate(RE_vs_RHE=0.715, A_el=0.196)
+    >>> my_ecms_meas.calibrate(ms_cal_results=[my_H2_at_M2, my_O2_at_M32])
+    >>> my_ecms_meas.calibrate(RE_vs_RHE=0.656)   # overshadows the first RE_vs_RHE
+    >>> my_ecms_meas.plot(mol_list=["H2", "O2"])
+
+  3. You can calibrate all at once.
+
+    >>> my_ecms_meas = my_ec_meas + my_ms_meas
+    >>> my_ecms_meas.calibrate(
+    >>>     RE_vs_RHE=0.715, A_el=0.196, ms_cal_results=[my_H2_at_M2, my_O2_at_M32]
+    >>> )   # note that all of these are keyword arguments to ECMSCalibration.
+    >>> my_ecms_meas.plot(mol_list=["H2", "O2"])
```

Comment:
Looks great and I understand the mechanism better now. In respect to my question the answer then is, if I understand it correct, that concatenating measurements with different calibrations makes no sense, so we will simple use the one that ends up the latest for any given parameter